components/Carousel/Carousel.scss

```diff
@@ -0,0 +1,48 @@
+@import 'topcoder/tc-includes';
+
+$pager-bg-color: #737380;
+
+.Carousel {
+  display: flex;
+  flex-direction: row;
+  &.hidden {
+    // visibility: hidden;
```

Comment:
Is this part a work in progress? If not, we can delete the `&.hidden` section here